webroot/src/store/user/user.actions.ts

```diff
@@ -199,4 +199,31 @@ export default {
             localStorage.removeItem(tokens[authType][key]); // Always remove localStorage to reduce edge cache states; e.g. from switching auth storage
         });
     },
+    getPrivilegePurchaseInformationRequest: ({ commit, dispatch }) => {
+        commit(MutationTypes.GET_PRIVILEGE_PURCHASE_INFORMATION_REQUEST);
+        return dataApi.getPrivilegePurchaseInformation().then((privilegePurchaseData) => {
+            dispatch('getPrivilegePurchaseInformationSuccess', privilegePurchaseData);
+            return privilegePurchaseData;
+        }).catch((error) => {
+            dispatch('getPrivilegePurchaseInformationFailure', error);
+        });
+    },
+    getPrivilegePurchaseInformationSuccess: ({ dispatch, commit, state }, privilegePurchaseData) => {
+        if (privilegePurchaseData?.compactCommissionFee?.compact === state?.currentCompact?.type) {
```

Comment:
Why would a compact compare to a "type"? If these are both one of `aslp`, etc., perhaps we can rename `compact` to `compactType` for naming consistency?